### PR TITLE
Updated Norwegian translation

### PR DESCRIPTION
**Pull request by matsbst**
Fixed typos and improved some translations with correct wording

### CODE REVIEW

File: ghost/i18n/locales/no/portal.json [hunk 1]
```diff
@@ -41,21 +41,21 @@
...
     "Edit": "Rediger",
-    "Email": "Epost",
-    "Email newsletter": "Epostbasert nyhetsbrev",
-    "Email preferences": "Innstillinger for epost",
-    "Emails": "Epost",
-    "Emails disabled": "Epost deaktivert",
+    "Email": "E-post",
+    "Email newsletter": "E-postbasert nyhetsbrev",
+    "Email preferences": "Innstillinger for e-post",
+    "Emails": "E-post",
```
Comment (mike182uk) [inline ghost/i18n/locales/no/portal.json:54]: This should be plural. Would `E-poster` be correct here?

File: ghost/i18n/locales/no/portal.json [hunk 2]
```diff
@@ -41,21 +41,21 @@
...
+    "Emails disabled": "E-post deaktivert",
```
Comment (mike182uk) [inline ghost/i18n/locales/no/portal.json:55]: This should be plural. Would `E-poster deaktivert` be correct here?

File: ghost/i18n/locales/no/portal.json [hunk 3]
```diff
@@ -92,20 +92,20 @@
...
     "Plan checkout was cancelled.": "Påmelding til plan ble kansellert. ",
     "Plan upgrade was cancelled.": "Oppgradering ble kansellert.",
     "Please contact {{supportAddress}} to adjust your complimentary subscription.": "Vennligst kontakt {{supportAddress}} for å justere ditt gratis abonnement.",
-    "Please enter {{fieldName}}": "",
-    "Please fill in required fields": "Vennligst fyll in påkrevde felt",
+    "Please enter {{fieldName}}": "Vennligst oppgi {{fieldName}}",
+    "Please fill in required fields": "Vennligst fyll inn påkrevde felt",
     "Price": "Pris",
-    "Re-enable emails": "Re-aktiver epost",
+    "Re-enable emails": "Re-aktiver e-post",
```
Comment (mike182uk) [inline ghost/i18n/locales/no/portal.json:103]: This should be plural. Would `Re-aktiver e-poster` be correct here?

**Comment by mike182uk** [comment]
Thanks for this @matsbst 🙏

I've left a couple of comments if you could take a look when you get a spare moment 👍

**Comment by sagzy** [comment]
> Thanks for this @matsbst 🙏
> 
> I've left a couple of comments if you could take a look when you get a spare moment 👍

Hey @matsbst, just making sure the PR review wasn't missed :)

**Review by matsbst — COMMENTED** [review]
Fixed some typos and new strings

**Comment by matsbst** [comment]
Should be fixed now 🙏🏻

**Comment by cathysarisky** [comment]
Hey @matsbst , a heads-up that I just wrapped some missing strings (including month and year) in portal.  If you have time to do a few more translations, that'd be awesome! :)   (There's also a new search.json with a few MORE new strings, to get the search app translated.)  If you're working in Github on the web, you'll go to your fork, make sure you're on the patch-2 branch, and click the 'sync fork' and 'update branch'.  That'll get you the new strings into your fork.

**Pull request merged**